src/main/java/seedu/address/logic/parser/delivery/DeliveryEditCommandParser.java

```diff
@@ -62,18 +73,15 @@ public DeliveryEditCommand parse(String args) throws ParseException {
                     .parseDeliveryDate(argMultimap.getValue(PREFIX_DATE).get()));
         }
         if (argMultimap.getValue(PREFIX_STATUS).isPresent()) {
-            deliveryEditDescriptor.setStatus(ParserUtil
-                    .parseDeliveryStatus(argMultimap.getValue(PREFIX_STATUS).get()));
+            deliveryEditDescriptor.setStatus(ParserUtil.parseDeliveryStatus(argMultimap.getValue(PREFIX_STATUS).get()));
```

Comment:
May want to separate `ParserUtil.parseDeliveryStatus(...)` into a separate variable such as `DeliveryStatus parsedStatus = ParserUtil.parseDeliveryStatus(...)`

so that you just have to `deliveryEditDescriptor.setStatus(parsedStatus)`